internal/pkg/service/stream/definition/sink_table.go

```diff
@@ -7,15 +7,24 @@ import (
 )
 
 const (
-	SinkTypeTable = SinkType("table")
+	SinkTypeTable    = SinkType("table")
+	TableTypeKeboola = TableType("keboola")
 )
 
+type TableType string
+
 // TableSink configures destination table.
 type TableSink struct {
-	Keboola TableSinkKeboola `json:"keboola"`
-	Mapping table.Mapping    `json:"mapping"`
+	Type    TableType     `json:"type" validate:"required,oneof=keboola"`
```

Comment:
Added `TableSink.Type`, so in the future, other types of target table, different from `keboola` can be implemented.

Reply:
In some next PR is `plugin system`, it builds on this loosely coupling between individual service parts.